src/halmos/sevm.py

```diff
@@ -844,7 +845,7 @@ def sha3_data(self, data: Bytes) -> Word:
             )
             sha3_expr = f_sha3(data)
         else:
-            sha3_expr = BitVec("f_sha3_0", BitVecSort256)
+            sha3_expr = EMPTY_KECCAK
```

Comment:
I don't see a drawback for this, we know the value for `f_sha3_0()`! We know it's not 0, and we also know it's the only keccak with that specific value.